### PR TITLE
Created tables, attempted functionality for location and weather

### PR DESCRIPTION
**Pull request by RazorWire13**

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+CREATE TABLE IF NOT EXISTS yelps {
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255),
+    image_url VARCHAR(255),
+    price CHAR(5),
+
+
+
+
+}
```
Comment (kowserk7) [inline schema.sql:29]: You have curly braces here when they should be parenthesis and a semi colon.

File: server.js [hunk 2]
```diff
@@ -4,42 +4,167 @@
...
+function Weather(day) {
+  this.tableName = 'weathers';
+  this.time = new Date(day.time * 1000).toString().slice(0, 15);
+  this.forecast = day.summary;
+  this.created_at = Date.now();
+}
+
+Weather.prototype = {
+  save: function (location_id) {
+    const SQL = `INSERT INTO ${this.tableName} (forecast, time, location_id) VALUES ($1, $2, $3, $4);`;
```
Comment (kowserk7) [inline server.js:78]: missing the created at column.

File: server.js [hunk 3]
```diff
@@ -102,6 +219,3 @@ function Movies(movies) {
   this.image_url = movies.poster_path;
```
Comment (kowserk7) [inline server.js:219]: missing 'https://image.tmdb.org/t/p/w200_and_h300_bestv2' + before the image path.

Comment (RazorWire13) [reply server.js:219]: Thank you, Koko!

**Review by kowserk7 — COMMENTED** [review]
Once you make those changes you can resubmit for full credit.

**Pull request merged**